### PR TITLE
Content changes to product pages

### PR DESCRIPTION
**Pull request by bethwalton**
Small changes for clarity and style.

### CODE REVIEW

File: source/about-govwifi/connect-to-govwifi.html.erb [hunk 1]
```diff
@@ -13,34 +13,34 @@ description: Use your public sector staff email address to connect devices to Go
...
+        <p class="govuk-body">You need to:</p>
         <ul class="govuk-list govuk-list--bullet">
-          <li>send a blank email to <%= link_to "signup@wifi.service.gov.uk", "mailto:signup@wifi.service.gov.uk", class: "govuk-link" %></li>
+          <li>send a blank email from your public sector email address to <%= link_to "signup@wifi.service.gov.uk", "mailto:signup@wifi.service.gov.uk", class: "govuk-link" %>:</li>
           <li>leave the subject line blank</li>
         </ul>
-        <p class="govuk-body">We will email your username and password confirmation in reply.</p>
-        <p class="govuk-body">If you do not receive the username and password confirmation e-mail within a few minutes of signing up, please check your spam/junk e-mail folder.</p>
+        <p class="govuk-body">We'll reply with your username and password.</p>
+        <p class="govuk-body">Check your spam or junk email folder if the email does not arrive after a few minutes.</p>
```
Comment (joelanman) [inline source/about-govwifi/connect-to-govwifi.html.erb:33]: Is it helpful to front-load 'if' clauses, like in the original, so people can know straight away if something applies to them?

File: source/support/check-organisation-email-address.html.erb [hunk 2]
```diff
@@ -11,19 +11,18 @@ description: Check your government or public sector email address to sign up to
...
         </p>
 
         <div class="govuk-inset-text">
-          If your public sector staff email address is currently ineligible, please <a href="https://admin.wifi.service.gov.uk/help/new/user_support" class="govuk-link">contact us</a> to register your public sector organisation email domain.
+          If you do not have a public sector email address, you can
+            <%= link_to "connect to GovWifi as a visitor", "support/visitor-access-to-govwifi", class: "govuk-link" %>.
         </div>
 
-        <p class="govuk-body">You can check to see if your public sector staff email address is eligible below:</p>
+        <p class="govuk-body"> Check this list to see if your email address is eligible. <a href="https://admin.wifi.service.gov.uk/help/new/user_support" class="govuk-link">Let us know</a> if your public sector email address is not here.
```
Comment (joelanman) [inline source/support/check-organisation-email-address.html.erb:23]: you have to be signed in to see this, we could link to the public one instead?

File: source/support.html.erb [hunk 3]
```diff
@@ -12,28 +12,28 @@ description: Help and support connecting to GovWifi
...
+          If you need a GovWifi username or password reminder, send a blank email from the email address you registered with to <%= link_to "signup@wifi.service.gov.uk", "mailto: signup@wifi.service.gov.uk", class: "govuk-link" %>
         </p>
         <h4 class="govuk-heading-s">Check GovWifi service status</h4>
         <p class="govuk-body">
-          You can check the <%= link_to "status of our central authentication service", "https://status.wifi.service.gov.uk", class: "govuk-link" %>.
+          If you're having trouble connecting to GovWifi, check our <%= link_to "status page", "https://status.wifi.service.gov.uk", class: "govuk-link" %> to see if there are any problems.
         </p>
         <p class="govuk-body">
-          If our service is operating normally and you are still having difficulty connecting to GovWifi, please contact the IT support desk in your building.
+          If there are not any problems, contact IT support in the building you're in.
```
Comment (joelanman) [inline source/support.html.erb:32]: suggest: 'If there are no problems' or maybe 'If there are no problems on the status page'

File: source/about-govwifi/connect-to-govwifi.html.erb (hunk 1 above)
Comment (bethwalton) [reply source/about-govwifi/connect-to-govwifi.html.erb:33]: Yep can do

File: source/support.html.erb (hunk 3 above)
Comment (bethwalton) [reply source/support.html.erb:32]: 👍

File: source/support/check-organisation-email-address.html.erb (hunk 2 above)
Comment (bethwalton) [reply source/support/check-organisation-email-address.html.erb:23]: Argh yes, course.

File: source/about-govwifi.html.erb [hunk 4]
```diff
@@ -13,29 +13,28 @@ description: GovWifi is an authentication service that makes all participating g
...
         </p>
         <p class="govuk-body">
-          GovWifi allows public sector staff and visitors to signup once and use
+          With GovWifi, public sector staff and visitors only need to sign up once. They can then use
           the same username and password to automatically connect to a single wifi service
           across multiple public sector locations.
         </p>
         <div class="govuk-inset-text">
-          <strong><%= link_to "Learn more about how to connect your device to GovWifi", "/about-govwifi/connect-to-govwifi", class: "govuk-link" %></strong>
+          See how to<%= link_to "connect your device to GovWifi", "/about-govwifi/connect-to-govwifi", class: "govuk-link" %>
```
Comment (joelanman) [inline source/about-govwifi.html.erb:37]: needs a space after 'how to'

File: source/support/check-organisation-email-address.html.erb (hunk 2 above)
Comment (joelanman) [reply source/support/check-organisation-email-address.html.erb:23]: ugh this was weird. You don't have to be signed in. But! If you were signed in, and then timed out, you do get asked to sign in.

File: source/about-govwifi/connect-to-govwifi.html.erb (hunk 1 above)
Comment (bethwalton) [reply source/about-govwifi/connect-to-govwifi.html.erb]: Done

File: source/about-govwifi.html.erb (hunk 4 above)
Comment (bethwalton) [reply source/about-govwifi.html.erb]: Done

File: source/support/check-organisation-email-address.html.erb (hunk 2 above)
Comment (bethwalton) [reply source/support/check-organisation-email-address.html.erb:23]: We decided to leave it as it is as we're not aware it's causing users any issues, but we'll be investigating this as part of a wider look at our support pages.

**Review by freesteph — DISMISSED** [review]
I like it.

**Review by joelanman — APPROVED** [review]
looks good!

**Review by joelanman — APPROVED** [review]
Looks good!

**Review by freesteph — APPROVED** [review]

**Pull request merged**